core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1973,7 +2067,12 @@ class SparkContext(config: SparkConf) extends Logging {
   }
 
   /**
-   * Run a job on all partitions in an RDD and pass the results to a handler function.
+   * Run a job on all partitions in an RDD and pass the results to a handler function, but take
+   * a function of type `Iterator[T] => U` instead of `(TaskContext, Iterator[T]) => U`.
```

Comment:
I think this one should change too -- instead of talking about the types (which are already in the signature) refer to the salient difference here, that this one doesn't take a function that takes a TaskContext.